src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  gap: 1px;
+  width: $day-size * $days-in-week + 20px;
+  height: 100vh;
+  padding: 10px;
+
+  &__cell {
+    box-sizing: border-box;
+    width: $day-size;
+    height: $day-size;
+    position:relative;
+    transition-duration: 0.5s;
+
+    &:hover {
+      cursor: pointer;
```

Comment:
Don't set `cursor` in hover

Reply:
it was stated in readme:
On hovering over a cell, the cursor should become pointer.